contracts/ethereum/src/PaymentRegistry.sol

```diff
@@ -128,8 +109,6 @@ contract PaymentRegistry is Initializable, OwnableUpgradeable, UUPSUpgradeable {
             new bytes[](0), //factory dependencies
             msg.sender //refund recipient
         );
-
-        emit ClaimPayment(transferInfo);
```

Comment:
Why did u remove the emit?

Reply:
No one is currently reading this event, so I removed to save on Gas.

Do you think it is a necessary event? Who would need to read it?

Reply:
It was decided the Event must remain for debugging, i have added it back